tests/running/math.lua

```diff
@@ -225,6 +227,9 @@ function test18()
            math.min(3, 3.0, 3.0) == 3 and
            is_integer(math.min(3, 3.0, 3.0)) and
            math.min(5, 4, 3, 2, 1, 0, -10, -9, -8, -7, -6, -5, -4, -3, -2, -1) == -10
+    -- Tests we do not currently pass due to incompatibility with PUC-Rio Lua
+    --     is_err(math.min("1"))
+    --     is_nan(math.min(0.0 % 0.0, 1, 2))
```

Comment:
I'm pretty sure `math.min` with a nan argument is pretty much undefined, as `math.min(nan, 1.0)` and `math.min(1.0, nan)` are different in PUC-Rio Lua.

Reply:
They are different.  The code matches behavior with `math.min(1.0, nan)` but not with `math.min(nan, 1.0)`